apps/app/src/app/pages/pages/sessions/pages/session-detail/session-detail.component.ts

```diff
@@ -33,11 +35,15 @@ export class SessionDetailComponent implements OnInit {
     private route: ActivatedRoute,
     private browserTitleService: BrowserTitleService,
     private languageService: LanguageService,
-    private tasksService: TasksService
+    private tasksService: TasksService,
+    public autoRefreshService: AutoRefreshService
   ) {
     this.browserTitleService.setTitle(
       this.languageService.instant('pages.sessions.session-detail.title')
     );
+
+    // Activate auto refresh
+    this.autoRefreshService.setFn(() => this.onManualTasksRefresh());
```

Comment:
call directly `onTaskRefresh` ?

`onTaskRefresh`should most likely have an optional argument ( or nullable).

Make it consistent with `apps/app/src/app/pages/pages/sessions/sessions.component.ts`

Reply:
done